### PR TITLE
Add disableInviteNotifications option to drop invitations notifs

### PR DESCRIPTION
**Pull request by fflorent**
This option is complementary to disableJoinLeaveNotifications. A matrix bot may send notifications about users being invite in the room (e.g. IRC users). But we don't want Discord users to be notified (because that creates noises in the discussions).

Also I wonder if the `disableJoinLeaveNotifications` and `disableInviteNotifications` are not redundant in term of use case (I mean I am not sure if users would use different values for both options).

### CODE REVIEW

File: test/test_config.ts [hunk 1]
```diff
@@ -61,9 +63,11 @@ describe("DiscordBridgeConfig.applyConfig", () => {
         config.applyEnvironmentOverrides({
             APPSERVICE_DISCORD_BRIDGE_DISABLE_DELETION_FORWARDING: false,
             APPSERVICE_DISCORD_BRIDGE_DISABLE_JOIN_LEAVE_NOTIFICATIONS: true,
+            APPSERVICE_DISCORD_BRIDGE_DISABLE_INVITE_NOTIFICATIONS: true,
```
Comment (fflorent) [inline test/test_config.ts]: Not sure how that works, but the test seems to pass on my computer.

**Comment by Sorunome** [comment]
Lint failed, would you please fix that? (`npm run lint`). Other than that it looks good!

**Comment by fflorent** [comment]
@Sorunome fixed, thanks!

**Comment by Half-Shot** [comment]
I wonder why the formatting tests are failing :thinking:

**Comment by fflorent** [comment]
@Half-Shot Oh! Is this something I am supposed to fix?

**Comment by auscompgeek** [comment]
It looks like this PR is based on master. Needs to be based on develop, methinks.

**Comment by fflorent** [comment]
@auscompgeek fixed, good catch, thanks!

**Comment by Sorunome** [comment]
This looks good, thank you!

**Pull request merged**